Content.Tests/DMTests.cs

```diff
@@ -176,6 +198,9 @@ private static DMTestFlags GetDMTestFlags(string sourceFile) {
         return testFlags;
     }
 
+    [GeneratedRegex(@"OD.{4}")]
```

Comment:
Should probably just accept numbers
```suggestion
    [GeneratedRegex(@"OD[0-9]{4}")]
```